### PR TITLE
Add API to access config db

### PR DESCRIPTION
**Pull request by taoyl-ms**

### CODE REVIEW

**Comment by lguohan** [comment]
:shipit:

File: src/swsssdk/configdb.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        Args:
+            table: Table name.
+        Returns: 
+            Table data in a dictionary form of 
+            { 'row_key': {'column_key': 'value', ...}, ...}
+        """
+        pattern = '{}:*'.format(table.upper())
+        keys = SonicV2Connector.keys(self, self.CONFIG_DB, pattern)
+        data = {}
+        for key in keys:
```
Comment (lguohan) [inline src/swsssdk/configdb.py]: what is the behavior when the table is not created?

Comment (lguohan) [inline src/swsssdk/configdb.py]: do you throw exception or return empty list.

File: src/swsssdk/configdb.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        Args:
+            table: Table name.
+            key: Key of table row.
+            data: Table row data in a form of dictionary {'column_key': 'value', ...}
+        """
+        client = self.redis_clients[self.CONFIG_DB]
+        _hash = '{}:{}'.format(table.upper(), key)
+        client.hmset(_hash, data)
+
+    def get_table(self, table, key):
```
Comment (lguohan) [inline src/swsssdk/configdb.py]: get_entry

File: src/swsssdk/configdb.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+    def __init__(self):
+        """Connect to Redis through TCP, which does not requires root.
+        """
+        super(ConfigDBConnector, self).__init__(host='127.0.0.1')
+
+    def connect(self):
+        SonicV2Connector.connect(self, self.CONFIG_DB, False)
+
+    def set_table(self, table, key, data):
```
Comment (lguohan) [inline src/swsssdk/configdb.py]: set_entry

File: src/swsssdk/configdb.py [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        Args:
+            table: Table name.
+            key: Key of table row.
+        Returns: 
+            Table row data in a form of dictionary {'column_key': 'value', ...}
+        """
+        _hash = '{}:{}'.format(table.upper(), key)
+        return SonicV2Connector.get_all(self, self.CONFIG_DB, _hash)
+
+    def get_table_all(self, table):
```
Comment (lguohan) [inline src/swsssdk/configdb.py]: get_table

File: src/swsssdk/configdb.py (hunk 1 above)
Comment (taoyl-ms) [inline src/swsssdk/configdb.py]: Return an empty dictionary. See comments.

---
In reply to: [127306170](https://github.com/Azure/sonic-py-swsssdk/pull/7#discussion_r127306170) [](ancestors = 127306170)

**Pull request merged**